scripts/handlebars/templates/transforms.hbs

```diff
@@ -25,7 +25,7 @@ There are 3 types of transforms: attribute, name, and value.
 
 **Name:** A name transform transform the name of a property. You should really only be apply one name transformer because they will override each other if you use more than one.
 
-**Value:** The value transform is the most important as this is the one that changes the representation of the value. Colors can be turned into hex values, rgb, hsl, hsv, etc. Value transforms have a matcher function so that they only get run on certain properties. This allows us to only run a color transform on just the colors and not every property.
+**Value:** The value transform is the most important as this is the one that changes the representation of the value. Colors can be turned into hex values, rgb, hsl, hsv, etc. Value transforms have a matcher function by which they only get run on certain properties. This allows us to only run a color transform on only the colors and not every property.
```

Comment:
> **Value:** The value transform is the most important as this is the one that changes the representation of the value. Colors can be turned into hex values, rgb, hsl, hsv, etc. Value transforms have a matcher function by which they only get run on certain properties. This allows us to only run a color transform on only the colors and not every property.

**Value:** The value transform is the most important as this is the one that changes the representation of the value. Colors can be turned into hex values, rgb, hsl, hsv, etc. Value transforms have a matcher function which ensure they only get run on certain properties. This allows us to run a color transform on only the colors and not every property.